src/sparseml/pytorch/utils/exporter.py

```diff
@@ -571,6 +571,18 @@ def export_onnx(
     onnx.save(onnx_model, file_path)
 
     if convert_qat and is_quant_module:
+        if (
+            _PARSED_TORCH_VERSION.major == 1
+            and _PARSED_TORCH_VERSION.minor in [10, 11]
+            and "NM_FORCE_QUANT_EXPORT" not in os.environ
+        ):
+            raise RuntimeError(
+                "Quantized exports are not supported in torch==1.10.* or 1.11.*. "
+                f"Found torch version {torch.__version__}. "
+                "To override this error, set environment variable "
+                "`NM_FORCE_QUANT_EXPORT` to anything."
```

Comment:
The condition would pass if `NM_FORCE_QUANT_EXPORT` is set to anything, but could we specify a value, like `NM_FORCE_QUANT_EXPORT=true` or `1` to make it a less confusing for users? Does that make sense?

Reply:
Updated